src/engine/plugins/table.cpp

```diff
@@ -106,6 +132,30 @@ Status TablePlugin::HandleRequest(const RoutingAlgorithmsInterface &algorithms,
             {
                 const auto &table_index = row * num_destinations + column;
                 BOOST_ASSERT(table_index < result_tables_pair.first.size());
+                // Add decel/accel penalty to account for stoppage at the start/end of trip
+                if (stoppage_penalty_range >= 0 &&
+                    result_tables_pair.first[table_index] != MAXIMAL_EDGE_DURATION)
+                {
+                    const auto &source =
+                        snapped_phantoms[params.sources.empty() ? row : params.sources[row]];
+                    const auto &destination =
+                        snapped_phantoms[params.destinations.empty() ? column
+                                                                     : params.destinations[column]];
+
+                    // TODO: if phantom node is actual node, will the distance/duration be 0?
```

Comment:
can anyone with more experience speak to this question? when the correlated point is close to a graph node, can the distance/duration be really small? do i need to worry about `reverse_distance/duration`?